reference/docs-conceptual/PowerShell-Core-Support.md

```diff
@@ -0,0 +1,68 @@
+# PowerShell Core Support Lifecycle
+
+PowerShell Core is a distinct set of tools and components that ships separately from Windows PowerShell.
+Therefore, PowerShell Core is not included in the Windows 10 or Windows Server licensing agreements.
+
+However, PowerShell Core is supported under traditional Microsoft support agreements, including [Premier][] and [Microsoft Enterprise Agreements][enterprise-agreement].
```

Comment:
We should mention that only some distributions of PowerShell Core are supported this way

Reply:
Should we mention SA (Software Assurance)?

Reply:
Good feedback @TravisEz13 

@iSazonov I'm actually not familiar with *all* of the possible support agreements. Do you have a link handy for SA offerings by any chance?

Reply:
Never mind, I found it. :)

Let me talk to people on my side, make sure it's appropriate.